cas-management-webapp/src/main/java/org/jasig/cas/services/web/beans/RegisteredServiceEditBean.java

```diff
@@ -82,35 +83,80 @@
  * @since 4.1
  */
 public final class RegisteredServiceEditBean implements Serializable {
+    /**
+     * The constant serialVersionUID.
+     */
     private static final long serialVersionUID = 4882440567964605644L;
 
+    /**
```

Comment:
We dont require them; most have been there from before. The ones that we have written could be useless at times and too obvious, but I think we can do better. We just bad docs, not that docs are useless. That said, I think we can relax them further.